### ISSUE TITLE
Ajax Bug

### ISSUE THREAD

**Issue by Alf21** [adeptoas/sweet-modal#10]
If using Ajax, this plugin 'crashes':

The backdrop (modal effect) wont disappear and the body still behave with overflow: hidden; ...
This happens in the following code:

```
      $.sweetModal.prompt('Why do you want to delete this picture?', 'Because...', 'Because it is doubled (#XXXX).', function(reason) {
        $.sweetModal('Test.');
        reason = reason.trim();
        if(reason && reason.length > 0) {
          suggestDelete(id, reason, { // AJAX request using PHPLiveX
            "onFinish": function(response, root) {
        	    $.sweetModal('Your suggestion was transmitted to the team.');
            }
          });
        }
      });
```

The Popup 'Test.' will appear after clicking ok but never the second one

**Comment by bluefirex** [comment]
This is related to https://github.com/adeptoas/sweet-modal/issues/9

**Comment by bluefirex** [comment]
Fixed in edc7e512ea16a354e32034d25eadf22052ca25b4, released in [v1.3.3](https://github.com/adeptoas/sweet-modal/releases/tag/v1.3.3)

**Comment by Alf21** [comment]
Nice, everything works fine. Good job!
